### DISCUSSION TITLE
支持链接元素 target 属性

### DISCUSSION THREAD

**Discussion by bymoye** [PaperStrike/Pjax#83] [Ideas]
目前对<a>标签中带有target="_blank"节点的也会默认进行pjax跳转 
但我认为这是不合理的 
因为这个属性本义就是打开新的窗口.
虽然最后也会正常跳转 但是也多了一个fetch请求

**Comment by PaperStrike** [comment]
#84 ，在 1.0.0 时发布。

有关触发 Pjax 跳转的**某些元素属性的支持** 多数会被考虑成新特性，不一定会像 BUG 一样解决，比如 `<a>` 和 `<area>` 的 `ping` 属性大概率是不会支持的。这些时候你需要考虑 自己调用 [`loadURL`](https://github.com/PaperStrike/Pjax/tree/v1.1.0#loadurl) 或 [`weakLoadURL`](https://github.com/PaperStrike/Pjax/tree/v1.1.0#weakloadurl)，关掉 [`defaultTrigger`](https://github.com/PaperStrike/Pjax/tree/v1.1.0#defaulttrigger) 这样子。
